hpat/tests/test_utils.py

```diff
@@ -53,3 +53,6 @@ def get_start_end(n):
     start = hpat.distributed_api.get_start(n, n_pes, rank)
     end = hpat.distributed_api.get_end(n, n_pes, rank)
     return start, end
+
+def check_numba_version(version):
+    return numba.__version__ == version
```

Comment:
`numba.__version__` is shorter than `check_numba_version`
may be it is better to use this variable from Numba directly inplace?

Reply:
I don't know. For me both versions are fine, though version with function is a little bit better, since it doesn't looks so ugly and encapsulates details